actors/evm/src/interpreter/precompiles/bls_util.rs

```diff
@@ -0,0 +1,422 @@
+use super::PrecompileError;
+use blst::{
+    // Basic types
+    blst_bendian_from_fp,
+    blst_fp,
+    blst_fp_from_bendian,
+    blst_fp2,
+    blst_p1,
+    blst_p1_affine,
+    blst_p1_affine_in_g1,
+    blst_p1_affine_is_inf,
+    blst_p1_affine_on_curve,
+    blst_p1_from_affine,
+    blst_p1_mult,
+    blst_p1_to_affine,
+    blst_p2,
+    blst_p2_affine,
+    blst_p2_affine_in_g2,
+    blst_p2_affine_is_inf,
+    blst_p2_affine_on_curve,
+    blst_p2_from_affine,
+    blst_p2_mult,
+    blst_p2_to_affine,
+    blst_scalar,
+    blst_scalar_from_bendian,
+};
+
+pub const G1_INPUT_LENGTH: usize = 128;
```

Comment:
This should just be replaced with padded G1 length

Reply:
Yeah it looks like revm does it the same way I'm imagining: https://github.com/bluealloy/revm/blob/6697f70dd3165e5149a0342b29d6f23f7b082cac/crates/precompile/src/bls12_381_const.rs#L145